source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
+		result := this + This new(converted)
+		converted free()
+		result
+	}
+	operator + (other: UInt) -> This {
+		converted := other toString()
+		result := this + This new(converted)
+		converted free()
+		result
+	}
+	operator + (other: Float) -> This {
+		converted := other toString()
+		result := this + This new(converted)
+		converted free()
+		result
+	}
+	operator + (other: Double) -> This {
+		converted := other toString()
+		result := this + This new(converted)
+		converted free()
+		result
+	}
```

Comment:
Imagine how many lines an interface could have saved us in a situation like this...